src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/OtterResourceContainerConfig.kt

```diff
@@ -6,39 +6,33 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
 import com.fasterxml.jackson.module.kotlin.KotlinModule
 import org.wycliffeassociates.resourcecontainer.Config
-import java.io.File
-import java.io.IOException
+import java.io.BufferedReader
+import java.io.BufferedWriter
 
 class OtterResourceContainerConfig : Config {
-
     var config: OtterConfig? = null
     var extendedDublinCore: ExtendedDublinCore? = null
 
-    override fun read(configFile: File): Config {
-        if (configFile.exists()) {
-            val mapper = ObjectMapper(YAMLFactory())
-            mapper.registerModule(KotlinModule())
-            config = configFile.bufferedReader().use {
-                mapper.readValue(it, OtterConfig::class.java)
-            }
-            config?.let {
-                extendedDublinCore = it.extendedDublinCore
-            }
-            return this
-        } else {
-            throw IOException("Missing config.yaml")
+    override fun read(br: BufferedReader): Config {
```

Comment:
I think this needs to be updated due to the resource container change

Reply:
done